docs/kb/semgrep-ci/ci-vs-cli.md

```diff
@@ -44,11 +44,11 @@ would give similar results to `semgrep ci`.
 
 ## Pro analysis
 
-When using `semgrep ci` with Semgrep AppSec Platform, the Semgrep scan reflects the Pro Engine options configured in Semgrep AppSec Platform at https://semgrep.dev/orgs/-/settings. If Pro Engine is enabled, and the scanned code includes [supported Pro languages](/docs/supported-languages/#semgrep-code-language-support), then interfile and interprocedural analysis is performed.
+When using `semgrep ci` with Semgrep AppSec Platform, you can configure whether the scan uses cross-file analysis in [Settings](https://semgrep.dev/orgs/-/settings). If you enable cross-file analysis, Semgrep performs interfile and interprocedural analysis for [supported languages](/supported-languages#language-maturity-summary).
```

Comment:
@khorne3 This feels weird to me because we're switching between cross-file and interfile. I'm not totally clear on when we prefer one vs the other, but what about this:
```suggestion
When using `semgrep ci` with Semgrep AppSec Platform, you can configure whether the scan uses cross-file analysis in [Settings](https://semgrep.dev/orgs/-/settings). If you enable cross-file analysis, Semgrep performs cross-file and cross-function analysis for [supported languages](/supported-languages#language-maturity-summary).
```

Reply:
I did a quick search of the repo, and we prefer cross-file (though sometimes we'll do something like `cross-file (interfile)`) -- I like your suggestion to use `cross-file` here.